libpimeval/src/pimPerfEnergyAquabolt.cpp

```diff
@@ -7,7 +7,15 @@
 #include "pimPerfEnergyAquabolt.h"
 #include "pimCmd.h"
 #include <iostream>
+#include <cmath>
 
+// Aquabolt adds a SIMD FPU shared between two banks, with only one bank accessing it at a time.
+// The supported FPU instructions are: ADD, MUL, MAC, and RELU. However, RELU is currently not implemented in the simulator.
```

Comment:
Hmm, I was thinking if we could model subtraction using 2's complement but that won't be possible without modifying the FPU